support-workers/src/typescript/errors/zuoraErrors.ts

```diff
@@ -0,0 +1,35 @@
+import type { ZuoraError } from '@guardian/support-service-lambdas/modules/zuora/src/errors/zuoraError';
+import { retryLimited, retryNone } from './retryError';
+
+const transactionDeclinedMessages = [
```

Comment:
If there's a period where both the Scala version and the TypeScript are being used it'd be good to keep this list in sync. Maybe worth adding a comment along those lines?